### ISSUE TITLE
[Accessibility: ACC-270.6] Table: UI cannot be activated with keyboard.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12618]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
It must be possible to invoke/activate each & every interactive UI, using the keyboard.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Platform Table](https://sap.github.io/fundamental-ngx/#/platform/table/clickable-rows)
Within the table ‘Realms’ user is not able to activate any row using keyboard.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/ab567aee-489c-40d8-8017-0bfd25945dec">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 the screenshots shows an application. Can you create a stackblitz example so we can provide you with the right support?

**Comment by droshev** [comment]
closing as there was no activity
